packages/plugins/src/components/plugin-context/index.tsx

```diff
@@ -15,13 +25,13 @@ export { Provider as PluginContextProvider };
  * A Higher Order Component used to inject Plugin context to the
  * wrapped component.
  *
- * @param {Function} mapContextToProps Function called on every context change,
- *                                     expected to return object of props to
- *                                     merge with the component's own props.
+ * @param mapContextToProps Function called on every context change,
+ *                          expected to return object of props to
+ *                          merge with the component's own props.
  *
- * @return {WPComponent} Enhanced component with injected context as props.
```

Comment:
This `WPComponent` return type doesn't play happy -- way easier to let inference do its magic